### PR TITLE
Use jobname base to determine running image in a flink update.

### PR DESCRIPTION
**Pull request by nielsdenissen**
This is done to ensure that appending a version number to your jobname, still allows you to perform a rolling update. Even though the names don't match fully.

Also, some improvements on the docker setup are done with regards to the entrypoint.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 cover.out
+cli
```
Comment (mrooding) [inline .gitignore]: Missing newline at end of the file

File: cmd/cli/main.go [hunk 2]
```diff
@@ -75,11 +75,11 @@ func DeployAction(c *cli.Context) error {
 func UpdateAction(c *cli.Context) error {
 	update := UpdateJob{}
 
-	jobName := c.String("jobname")
-	if len(jobName) == 0 {
-		return cli.NewExitError("unspecified flag 'jobname'", -1)
+	jobNameBase := c.String("jobname-base")
```
Comment (mrooding) [inline cmd/cli/main.go]: Maybe `job-name-base` is a better param?

File: .gitignore (hunk 1 above)
Comment (nielsdenissen) [reply .gitignore]: Done

File: cmd/cli/main.go (hunk 2 above)
Comment (nielsdenissen) [reply cmd/cli/main.go]: At the moment in other places we also use jobname appended, like query command. Also some are camel and some are snake case. Shall I move all to Camel case (e.g. jobNameBase)?

Comment (mrooding) [reply cmd/cli/main.go]: We're still talking about the cmd line arguments right? I don't think that we should use camel case for the cmd line arguments. Dash separated seems to be the more widely used approach. using camelcase for variables itself is fine though.

Comment (nielsdenissen) [reply cmd/cli/main.go]: Alright, then I'll change all to dash separated (some lingering with camelcase atm)

**Pull request merged**